pkg/analysis_server/lib/src/services/correction/fix.dart

```diff
@@ -214,6 +214,7 @@ class DartFixKind {
       "Remove parentheses in getter invocation");
   static const REMOVE_THIS_EXPRESSION =
       const FixKind('REMOVE_THIS_EXPRESSION', 50, "Remove this expression");
+  static const REMOVE_TYPE = const FixKind('REMOVE_TYPE', 50, "Remove type");
```

Comment:
@bwilkerson what do you think about the message given to the user, does 'Remove type' still ok?